isos/appliance/nat-setup

```diff
@@ -40,32 +43,29 @@ echo 2 > /proc/sys/net/ipv4/ip_dynaddr
 # too many routers are still ignorant
 echo 0 > /proc/sys/net/ipv4/tcp_ecn
 
-# Set a known state
-iptables -P INPUT   ACCEPT
-iptables -P FORWARD ACCEPT
-iptables -P OUTPUT  ACCEPT
-
-# These lines are here in case rules are already in place and the
-# script is ever rerun on the fly. We want to remove all rules and
-# pre-existing user defined chains before we implement new rules.
-iptables -F
-iptables -X
-iptables -Z
-
-iptables -t nat -F
-
-# Allow local-only connections
-iptables -A INPUT  -i lo -j ACCEPT
-
-# Free output on any interface to any ip for any service
-# (equal to -P ACCEPT)
-iptables -A OUTPUT -j ACCEPT
-
-# Permit answers on already established connections
-# and permit new connections related to established ones
-# (e.g. port mode ftp)
-iptables -A INPUT -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT
+# wait for the bridge interface to exist - 100ms wait
+until ip link show dev bridge;do sleep 0.1;done
+
+iptables-restore <<RULES
```

Comment:
It doesn't. It needs to run after vic-init has renamed the interfaces which means it cannot run at the same time as iptables service (which is part of network-online).
It seemed less prone to confusion to rename the script and invoke it from a specific later target than to change the iptables service to depend on vic.target.